src/components/header/subheader/Subheader.test.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+import { render, screen, fireEvent } from "@testing-library/react";
+import { StoreProvider } from "../../../providers/StoreProvider";
+import { Button } from "../../shared/Button";
+
+describe("Activate modal", () => {
+  const setIsModalActive = jest.fn();
+  const buttonLabel = "Add new time entry";
+
+  it("Press button, setIsModalActive gets called", () => {
```

Comment:
We want these tests to be correct grammar wise and even more so describe the desired outcome of the test

```suggestion
  it("Presses the button, setIsModalActive gets called", () => {
```